log4j-core/src/test/java/org/apache/logging/log4j/core/CustomLevelsTest.java

```diff
@@ -26,6 +26,7 @@
 import org.apache.logging.log4j.Level;
 import org.apache.logging.log4j.junit.LoggerContextRule;
 import org.apache.logging.log4j.test.appender.ListAppender;
+
```

Comment:
We don't require whitespace here. We don't disallow it either, but not sure what the point is.